docs/src/content/docs/challenges/angular/4-context-outlet-typed.md

```diff
@@ -16,30 +16,34 @@ sidebar:
 
```

Comment:
@tomalaforge can i refactor the command in line 11 (expand this file here in github) to `angular-typed-context-outlet`? So it matches with the name of this challenge - Typed ContextOutlet.

Reply:
Yes you can if you want. Do it on a separate PR, It will be easier 👍